Ensure users can run more than one workspace (#1628)
codenvy/che-docs: modules/administration-guide/partials/con_persistent-volume-configuration.adoc

* The workspace PVC access mode must be reconfigured to `ReadWriteMany` (RWM), so multiple nodes can use this PVC simultaneously.

* Only one workspace in the same {orch-namespace} may be running. See xref:installation-guide:run-several-workspaces-at-a-time.adoc[].

> The `common` PVC strategy is not suitable for large multi-node clusters. Therefore, it is best to use it in single-node clusters. However, in combination with `per-workspace` {orch-namespace} strategy, the `common` PVC strategy is usable for clusters with around 75 nodes. The PVC used in this strategy must be large enough to accommodate all projects since there is a risk of the event, in which one project depletes the resources of others.

Reviewer: ```suggestion
The `common` PVC strategy is not suitable for large multi-node clusters. Therefore, it is best to use it in single-node clusters. However, in combination with the `per-workspace` {orch-namespace} strategy, the `common` PVC strategy is usable for clusters with not more than 75 nodes. The PVC used with this strategy must be large enough to accommodate all projects to prevent a situation in which one project depletes the resources of others.
```

* workspace logs
* additional Volumes defined by a user

> It’s a strategy when {prod-short} keeps its workspace data in assigned PVs that are allocated by a single PVC.

Reviewer: ```suggestion
With this strategy, {prod-short} keeps its workspace data in assigned PVs that are allocated by a single PVC.
```
